src/components/homepage.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Books from './Books';
+import './styles/home.css';
+import AddBookForm from './Addbook';
+
+const HomePage = () => (
+  <div className="container">
+    <header>
+      <ul className="links">
+        <li>
+          <h1> Bookstore CMS</h1>
+        </li>
+        <li>BOOKS</li>
+        <li>CATEGORIES</li>
+      </ul>
```

Comment:
- [ ] App should have 2 `<Link>`s in the navbar which leads to the booklist and categories locations. Kindly make the `<li>` links